README.md

```diff
@@ -96,7 +96,17 @@ So to trigger only the specific branch during the first time job configuration,
 #### Configure Global UI:
 Navigate to Dashboard → Manage Jenkins → System  
 Then go to the Synopsys Security Scan section.  
-And from there you can populate the inputs for configuration.
+And from there you can populate the inputs for configuration.     
+
+**Note:** To give input for secret fields such as password or token you need to use the jenkins credential provider. To do this follow these instructions,
+>- First you need to click on the add button. Then click Jenkins item on the dropdown.
```

Comment:
If user have already added secrets using (Manage Jenkins -> Credentials) will be listed in the drop down and user can select it.